service/src/test/java/org/apache/griffin/core/job/EventServiceTest.java

```diff
@@ -59,6 +61,9 @@ Licensed to the Apache Software Foundation (ASF) under one
     @Autowired
     private List<GriffinEvent> eventList;
 
+    @MockBean
```

Comment:
I don't know if the `IMetaStoreClient client` could be lazy initialized in `HiveMetaStoreProxy` ?
```
@Bean
    public IMetaStoreClient initHiveMetastoreClient() {
        HiveConf hiveConf = new HiveConf();
        hiveConf.set("hive.metastore.local", "false");
        hiveConf.setIntVar(HiveConf.ConfVars.METASTORETHRIFTCONNECTIONRETRIES,
            3);
        hiveConf.setVar(HiveConf.ConfVars.METASTOREURIS, uris);
        hiveConf.setIntVar(HiveConf.ConfVars.HMSHANDLERATTEMPTS, attempts);
        hiveConf.setVar(HiveConf.ConfVars.HMSHANDLERINTERVAL, interval);
        try {
            client = HiveMetaStoreClient.newSynchronizedClient(new HiveMetaStoreClient(hiveConf));
        } catch (Exception e) {
            LOGGER.error("Failed to connect hive metastore. {}", e);
        }
        return client;
    }
```
If not , `TestEntityManager` is been used in several test cases , Could we create a new  class to wrap it?

Reply:
> I don't know if the `IMetaStoreClient client` could be lazy initialized in `HiveMetaStoreProxy` ?

I'm not sure whether it works even the `IMetaStoreClient client` could be lazy initialized. I guess it will not work as `IMetaStoreClient client` still need be injected in `HiveMetaStoreServiceImpl` when using @DataJaTest.